pkg/reconcilers/deploymentconfig.go

```diff
@@ -57,6 +57,84 @@ func GenericBackendMutators() []DCMutateFn {
 	}
 }
 
+func DeploymentConfigListenerArgsMutator(_, existing *appsv1.DeploymentConfig) (bool, error) {
```

Comment:
I guess some unrelated content has been sneaked in.

Reply:
Yes this is from #862. I can rebase the [base feature branch](https://github.com/3scale/3scale-operator/tree/threescale-10163) off of master instead in order to make this PR's changes clearer. @eguzki just let me know which you'd prefer